build/configs/rtl8730e/loadable_apps_16M/defconfig

```diff
@@ -1209,8 +1209,8 @@ CONFIG_ELF_CACHE_READ=y
 CONFIG_ELF_CACHE_BLOCK_SIZE=2048
 CONFIG_ELF_CACHE_BLOCKS_COUNT=60
 # CONFIG_SYMTAB_ORDEREDBYNAME is not set
-# CONFIG_OPTIMIZE_APP_RELOAD_TIME is not set
-# CONFIG_SAVE_BIN_SECTION_ADDR is not set
+CONFIG_OPTIMIZE_APP_RELOAD_TIME=y
+CONFIG_SAVE_BIN_SECTION_ADDR=y
```

Comment:
1. To work mem leak, is this mandatory? If yes, we should check it on menuconfig using Kconfig dependency.
2. Let's enable them with new commit.

Reply:
1. yes , it is mandatory for loadable types . i didnot understand "we should check it on menuconfig using Kconfig dependecny"

Reply:
```
config MEM_LEAK  (I don't write exact name)
    depends on OPTIMIZE_APP_RELOAD_TIME  if LOADABLE
   or
   select OPTIMIZE_APP..._TIME if LOADABLE
```
You said they are mandatory to work memory leakage checker. Then, we should prevent them disabled with mem leak checker enabled.

Reply:
If the config is standalone (no dependency with other config), we can use `select`.
But if the config has other dependencies, then we can't use `select`. We should use `depends on`

Reply:
Mandatory is `CONFIG_SAVE_BIN_SECTION_ADDR` only? or both `CONFIG_SAVE_BIN_SECTION_ADDR` and `CONFIG_OPTIMIZE_APP_RELOAD_TIME`?

Reply:
> ```
> config MEM_LEAK  (I don't write exact name)
>     depends on OPTIMIZE_APP_RELOAD_TIME  if LOADABLE
>    or
>    select OPTIMIZE_APP..._TIME if LOADABLE
> ```
> 
> You said they are mandatory to work memory leakage checker. Then, we should prevent them disabled with mem leak checker enabled.

it is already implemented in Kconfig by using ```select```

Reply:
> Mandatory is `CONFIG_SAVE_BIN_SECTION_ADDR` only? or both `CONFIG_SAVE_BIN_SECTION_ADDR` and `CONFIG_OPTIMIZE_APP_RELOAD_TIME`?

Both